src/components/ai/SyntheticTherapyDemo.tsx

```
@@ -0,0 +1,599 @@
+                </TabsContent>
+              </Tabs>
+            </>
+          ) : (
+            <Card className="h-[400px] flex items-center justify-center">
+              <CardContent className="text-center">
+                <BrainIcon className="h-16 w-16 mx-auto mb-4 text-muted-foreground" />
+                <h3 className="text-xl font-medium mb-2">
+                  No Conversations Generated
+                </h3>
+                <p className="text-muted-foreground max-w-md">
+                  Configure the settings and click "Generate Conversations" to
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)